enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/AuthFilter.java

```diff
@@ -57,9 +58,21 @@ public boolean handleRequest(RequestContext requestContext) {
             }
         } catch (APISecurityException e) {
             //TODO: (VirajSalaka) provide the error code properly based on exception (401, 403, 429 etc)
-            requestContext.getProperties().put("code", "401");
-            requestContext.getProperties().put("error_code", e.getErrorCode());
-            requestContext.getProperties().put("error_description", e.getMessage());
+            Map<String, Object> requestContextProperties = requestContext.getProperties();
```

Comment:
Will this section be a common code we have to use for other filters aswell? if so this can be a util method